pkg/dotnet/core_deps/parse.go

```diff
@@ -19,6 +22,22 @@ func NewParser() types.Parser {
 	return &Parser{}
 }
 
+func splitNameVer(nameVer string) (name, version string) {
+	split := strings.Split(nameVer, "/")
+	if len(split) != 2 {
+		// Invalid name
+		log.Logger.Warnf("Cannot parse .NET library version from: %s", nameVer)
+		return "", ""
+	}
+	name = split[0]
+	version = split[1]
+	return
+}
+
+func joinNameVer(name, version string) (nameVer string) {
```

Comment:
You can rename this function and use it to create ID.
```suggestion
func packageID(name, version string) string {
```